client/packages/system/src/Patient/VaccinationCards/VaccineCardTable.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { FC, useEffect } from 'react';
```

Comment:
Mostly no changes in here, just copied from previous DetailView component. I'll put a comment on the bits that have changed.